components/ui/AppMegaDropdownMenu.vue

```diff
@@ -88,59 +240,68 @@ export default class TheMegaDropdownMenu extends Vue {
   color: $cool-gray-80;
   position: relative;
 
-  &__button {
-    &_primary,
-    &_secondary {
+  &__search-field-wrapper {
+    display: flex;
+    width: 18rem;
+    height: 2.5rem;
+    justify-content: space-between;
+    align-items: center;
+    padding: $spacing-03;
+
+    color: $cool-gray-80;
+    fill: $cool-gray-80;
+    border-bottom: 1px solid $purple-70;
+
+    &__input {
+      flex: 1;
       background-color: transparent;
       outline: none;
       border: none;
-      width: 18rem;
-      display: flex;
-      justify-content: space-between;
-      align-items: center;
-      padding: $spacing-03;
+      color: inherit;
+    }
+    &__input:focus::placeholder {
+      opacity: 0.25;
+    }
```

Comment:
We can nest this property:

```suggestion

      &:focus::placeholder {
        opacity: 0.25;
      }
    }
```

Reply:
good catch
fix: https://github.com/Qiskit/qiskit.org/pull/1823/commits/0930c160b15fa12eb26e69f1e57ab7b8b4e29730